action.yml

```diff
@@ -201,17 +201,11 @@ runs:
           echo "Running yarn build"
           yarn run build
           echo "Running git diff"
-          GIT_DIFF_FILES=$(git diff-files --ignore-all-space --relative=client)
-          GIT_DIFF=$(git diff --name-status --relative=client)
-          if [[ $GIT_DIFF_FILES != "" ]]; then
-            echo "git diff files:"
-            echo $GIT_DIFF_FILES
-          fi
+          git add .
```

Comment:
Why run the `git add` command? `git diff` will reveal changes that are not staged, `git diff --cached` will show diffs for those that are...

but if you're running both `git diff` and `git diff --cached` then what's the need to stage any changes?

Reply:
In case there were any dist files that for whatever reason were previously deleted and committed.  When rebuilding the dist files we need to git add them for them to show, otherwise they're untracked.

Probably don't need both git diff and git diff --cached though